grpc-native/src/main/java/org/ballerinalang/net/grpc/Message.java

```diff
@@ -130,9 +138,15 @@ public Message(
         }
 
         BMap<BString, Object> bBMap = null;
+        BArray bArray = null;
         if (type.getTag() == TypeTags.RECORD_TYPE_TAG) {
             bBMap = ValueCreator.createRecordValue(type.getPackage(), type.getName());
             bMessage = bBMap;
+        } else if (messageName.equals("Timestamp")) {
```

Comment:
Better to filter from tags rather than names.

Reply:
fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/312/commits/7ae50091a63cc9c98fe07cbc4284b9134d2a06d5